crypto/dsa/dsa.go

```diff
@@ -94,3 +94,14 @@ func PublicKeyToBytes(publicKey jwk.JWK) ([]byte, error) {
 		return nil, fmt.Errorf("unsupported key type: %s", publicKey.KTY)
 	}
 }
+
+func AlgorithmID(jwk *jwk.JWK) (string, error) {
+	switch jwk.KTY {
```

Comment:
this method is based on static assumptions of algorithms, which may be faulty. you can see the on-going discussion here https://github.com/TBD54566975/did-dht-method/issues/119

my recommendation is to keep this as-is until a conclusion is reached

Reply:
Thanks Gabe! created an issue to keep track https://github.com/TBD54566975/web5-go/issues/69